docs/reference/accessibility/orca-preferences/index.md

```diff
@@ -0,0 +1,65 @@
+(orca-preferences)=
+# Orca preferences
+
+Orca preferences allow you to customize functionality in Orca which applies to all applications. An example of an Orca preference is key echo because key echo is something that applies to all applications.
+
+Note that Orca preferences can be customized on an application-by-application basis. For instance you can set the default key echo to words and then set the key echo for Pidgin to be none. Having done so, Orca would always echo each word that you typed, unless you were in Pidgin. 
+
+Orca preferences can be configured globally or on the application level. Application-level preferences allow you to customize Orca functionality that only applies in certain environments, such as on web pages or in chat applications. As a result, you will only find these options available in the application-specific preferences dialogs and only for those applications to which these options apply. 
+
+## Keyboard shortcuts to open preferences dialogs
+
+*  {kbd}`Orca+Modifier`+ {kbd}`Space`: Orca's Preferences
```

Comment:
```suggestion
{kbd}`OrcaModifier+Space`
: Open Orca's Preferences
```

A couple of changes:

- Using a description list because this is a term-description structure.
- `Orca+Modifier` implied that you were supposed to press two keys, Orca and Modifier. Most accurately, the key is called `Orca Modifier`, but the `kbd` arguments can't contain spaces. In the same way, I'm using the `CapsLock` key.
- _Open_ Orca's preferences to clarify the action.

As an alternative, you could also consider the tabs for the two Orca layouts as used in how-to guides. Patrick had no complaints about them.